### ISSUE TITLE
getRuntime().isServerSide causes deploys to fail

### ISSUE THREAD

**Issue by bhainesva** [yext/pages#358]
### Describe the bug

I'm trying to use the `getRuntime().isServerSide` helper, but my publish is failing with this error:

```
Failed to generate page for feature index using entity 73998147 with locale en, in stream locations-r36qzvopmj, due to the following error: window is not defined:
at pA (PagesGenerator-5miwd637yc:src/assets/static/main-f562b1ed.js:92:6897)
at gA (PagesGenerator-5miwd637yc:src/assets/static/main-f562b1ed.js:92:7212)
at n (PagesGenerator-5miwd637yc:src/assets/render/_server.fabc5301.js:8:11357)
at Ke (PagesGenerator-5miwd637yc:src/assets/render/_server.fabc5301.js:8:12379)
at e.r.render (PagesGenerator-5miwd637yc:src/assets/render/_server.fabc5301.js:8:14950)
at e.r.read (PagesGenerator-5miwd637yc:src/assets/render/_server.fabc5301.js:8:14531)
at Object.z.renderToString (PagesGenerator-5miwd637yc:src/assets/render/_server.fabc5301.js:10:566)
at Module.et (PagesGenerator-5miwd637yc:src/assets/render/_server.fabc5301.js:14:41)
at N (PagesGenerator-5miwd637yc:src/assets/renderer/templateRenderer.js:22:682)
at K (PagesGenerator-5miwd637yc:src/assets/renderer/templateRenderer.js:22:3030)
```

### Reproduction

https://www.yext.com/s/3857853/yextsites/43839/branch/22940/deploy/5miwd637yc/details#tab=PAGE_GENERATION

### System Info

```shell
System:
    OS: macOS 12.3
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 148.15 MB / 32.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 17.4.0 - ~/.nvm/versions/node/v17.4.0/bin/node
    Yarn: 1.22.18 - /usr/local/bin/yarn
    npm: 8.3.1 - ~/.nvm/versions/node/v17.4.0/bin/npm
    pnpm: 7.9.0 - ~/.nvm/versions/node/v17.4.0/bin/pnpm
  Browsers:
    Chrome: 114.0.5735.198
    Edge: 114.0.1823.82
    Safari: 15.4
  npmPackages:
    @yext/pages: 1.0.0-beta.26 => 1.0.0-beta.23
```

### Used Package Manager

npm

### Used Browser

Chrome

### Validations

- [X] Check that there isn't [already an issue](https://github.com/yext/pages/issues) that reports the same bug to avoid creating a duplicate.

**Comment by mkilpatrick** [comment]
More context - https://yextops.slack.com/archives/C02UVSE7P6W/p1674849609241199?thread_ts=1674758407.088679&cid=C02UVSE7P6W

**Comment by asanehisa** [comment]
Fixed in #364
